### DISCUSSION TITLE
프로젝트 #1: 효율적으로 다양한 엔터프라이즈 테크를 확인 할수 있는 프레임워크

### DISCUSSION THREAD

**Discussion by bryantson** [Korean-American-IT-Association-KAITA/kaita-project-management#12] [3. 프로젝트 아이디어]
설명: 데브옵스에 배포 (deployment) 정보와 performance 정보를 align 시키는 프로젝트. 마이크로서비스 같은 기술을 사용해서 효율적인 배포를 목적. 엔터프라이즈 배포 시스템. 예를 들어 ISTIO 같은 서비스를 사용해서 deploy 했을때 performance 나 누가 했는지 알수가 없다. 하지만 데브옵스엔 이런게 가능. Audit, logging, monitoring 이 가능한 마이크로서비스. 각각의 당담하는 서비스는 있는데 종합적으로 하는건 없다. ISTIO, Jenkins, etc 등등에 있는걸 다 collection 해서 종합적으로 확인 하시는 오픈소스 서비스.

백그라운드: IBM Research POC

테크 스택: Harmonia - Framework, ISTIO, Jenkins, Kubernetes, GitHub, NLP

**Comment by Indosaram** [comment]
엄청 빡셀 것 같지만 재밌어보이네요. 제가 요즘 겪고 있는 문제라서 필요성에 공감합니다

**Reply by devminjae** [reply]
저도 개인적으로... 이 프로젝트는 따로라도 진행해보고 싶네요 ㅎ 재미있을것 같아요 ㅎ

**Reply by bryantson** [reply]
따로 진행도 가능합니다 ^^

**Reply by Indosaram** [reply]
한번 해보시죠ㅎㅎ 저는 in 입니다.

**Reply by jparksecurity** [reply]
이 프로젝트 재미있을 것 같습니다. 저도 참여해보고 싶습니다. 방장님 리포 하나 열어주실 수 있으실까요?

**Reply by haanmo** [reply]
해당 프로젝트의 논문입니다. 필요할 경우 미팅 시간에 더 설명하도록 하겠습니다.

https://www.researchgate.net/profile/Haan-Mo-Johng/publication/336821952_Harmonia_A_Continuous_Service_Monitoring_Framework_Using_DevOps_and_Service_Mesh_in_a_Complementary_Manner/links/5e7ebe87299bf1a91b828192/Harmonia-A-Continuous-Service-Monitoring-Framework-Using-DevOps-and-Service-Mesh-in-a-Complementary-Manner.pdf

**Reply by bryantson** [reply]
@jparksecurity 직접 여는게 안되나요? 궁금

**Reply by jparksecurity** [reply]
@bryantson  아 저는 회장님 허락을 받아야하나 해서요. 제가 리포 열 수 도 있군요. 아래에 링크걸었습니다.

https://github.com/Korean-American-IT-Association-KAITA/Continuous_Service_Monitoring

**Reply by bryantson** [reply]
@jparksecurity README.md 추가하시면 좋을것 같습니다
